src/Sample.Extension/SampleJobHostConfigurationExtensions.cs

```diff
@@ -33,6 +35,18 @@ public void Initialize(ExtensionConfigContext context)
                 context.Config.RegisterBindingExtensions(
                     new SampleAttributeBindingProvider(),
                     new SampleTriggerAttributeBindingProvider());
+
+                IConverterManager converterManager = context.Config.GetService<IConverterManager>();
+                converterManager.AddConverter<CloudTable, Table<OpenType>, TableAttribute>(typeof(CustomTableBindingConverter<>));
+            }
+        }
+
+        private class CustomTableBindingConverter<T>
```

Comment:
Allows the samples to run. I made the same change in the Twilio migration review. Will reconcile.